src/main/java/command/DeleteCommand.java

```diff
@@ -0,0 +1,50 @@
+package command;
+
+import fridayException.FridayException;
+import fridayException.InvalidDeleteArgument;
```

Comment:
I like that you specified the individual exception classes to be imported, instead of simply importing * from the fridayException package!